### PR TITLE
add some necessary description after running command 'minikube dashboard'

### LINKED ISSUE #26150: “Hello Minikube” tutorial runs dashboard subcommand in foreground
After issuing the command "minikube dashboard" in the "Create a minikube cluster" section, I never get the command prompt back. How, then, am I supposed to issue the commands listed in the following section, "Create a deployment"?

### PR DESCRIPTION
**Pull request by CKchen0726**
Fix #26150

### CODE REVIEW

**Comment by bradtopol** [comment]
Hi @sftim since you were involved with this can you serve as the reviewer?  Thanks!
/assign @sftim

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-26162--kubernetes-io-master-staging.netlify.app/docs/tutorials/hello-minikube/#create-a-minikube-cluster

File: content/en/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -55,10 +55,26 @@ If you installed minikube locally, run `minikube start`.
     minikube dashboard
     ```
 
+{{< note >}}
```
Comment (kbhawkey) [inline content/en/docs/tutorials/hello-minikube.md:58]: Hi @CKchen0726 . Thanks for contributing!
You could indent the `note` shortcode under the second list item and rephrase these sentences:

The `dashboard` command enables the dashboard add-on and opens the proxy in the default web browser. You can create Kubernetes resources on the dashboard such as Deployment and Service.

If you are running in an environment as root, see [Open Dashboard with URL](/docs/tutorials/hello-minikube#open-dashboard-with-url).

To stop the proxy, run `Ctrl+C` to exit the process. The dashboard remains running.

File: content/en/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -55,10 +55,26 @@ If you installed minikube locally, run `minikube start`.
...
+
 3. Katacoda environment only: At the top of the terminal pane, click the plus sign, and then click **Select port to view on Host 1**.
 
 4. Katacoda environment only: Type `30000`, and then click **Display Port**.
 
+## Open Dashboard with URL
+
+If you don't want to open a web browser, the dashboard command can also simply emit a URL:
+
+    ```shell
```
Comment (kbhawkey) [inline content/en/docs/tutorials/hello-minikube.md:74]: Would you remove the indentation of the command block (line 74)? See the preview page.

File: content/en/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -55,10 +55,26 @@ If you installed minikube locally, run `minikube start`.
...
+To stop the proxy (leaves the dashboard running), abort the started process (`Ctrl+C`).
+{{< /note >}}
+
 3. Katacoda environment only: At the top of the terminal pane, click the plus sign, and then click **Select port to view on Host 1**.
 
 4. Katacoda environment only: Type `30000`, and then click **Display Port**.
 
+## Open Dashboard with URL
+
+If you don't want to open a web browser, the dashboard command can also simply emit a URL:
```
Comment (kbhawkey) [inline content/en/docs/tutorials/hello-minikube.md:72]: nit: If you don't want to open a web browser, run the dashboard command with the `url` flag to emit a URL:

**Comment by kbhawkey** [comment]
Thanks @CKchen0726 .
/lgtm

**Pull request merged**